### PR TITLE
LPS-176412 Considering context path to build full URL

### PR DESCRIPTION
**Pull request by antonio-ortega**
Hi guys,

Can you review this pull request, please?

There is more information in LPS-176412, but basically this stopped working after [this commit](https://github.com/liferay/liferay-portal/commit/60f8d3aa4d4edfda40eb06b72e9fd81a14b7fb32).

Thanks!

Regards.

### CODE REVIEW

**Comment by antonio-ortega** [comment]
ci:test:sf

**Comment by 4lejandrito** [comment]
Thanks @antonio-ortega!

**Comment by 4lejandrito** [comment]
ci:forward

File: modules/apps/headless/headless-discovery/headless-discovery-impl/src/main/java/com/liferay/headless/discovery/internal/jaxrs/resource/v1_0/HeadlessDiscoveryOpenAPIResourceImpl.java [hunk 1]
```diff
@@ -255,7 +255,7 @@ private Map<String, List<String>> _getOpenAPIMap(String accept) {
 
 		String serverURL =
 			_portal.getPortalURL(_httpServletRequest) + _portal.getPathProxy() +
-				Portal.PATH_MODULE;
+				_portal.getPathContext() + Portal.PATH_MODULE;
```
Comment (ccorreagg) [inline modules/apps/headless/headless-discovery/headless-discovery-impl/src/main/java/com/liferay/headless/discovery/internal/jaxrs/resource/v1_0/HeadlessDiscoveryOpenAPIResourceImpl.java:258]: I think the `_portal.getPathProxy()` should be removed as the `pathContext()` method is already having into account the path proxy.

image

image

**Review by ccorreagg — CHANGES_REQUESTED** [review]
The proxy path would be duplicated

**Comment by antonio-ortega** [comment]
Thanks for catch it @ccorreagg !! 

Just sent a new commit removing proxy path.

**Review by ccorreagg — APPROVED** [review]
LGTM now!

**Comment by ccorreagg** [comment]
ci:forward

**Comment by ccorreagg** [comment]
Thanks @antonio-ortega for your work!

**Comment by yanan** [comment]
Tests failures are not related from qa perspective.

**Comment by ccorreagg** [comment]
Thanks @yanan !

**Comment by ccorreagg** [comment]
ci:forward:force

**Pull request closed without merging**